docker-compose/README.md

```diff
@@ -19,33 +19,33 @@ You can control the platform using the `-Dquarkus.docker.buildx.platform=linux/a
 
 Follow instructions under the [Script options](#script-options) section to use the locally built image.
 
-## Stargate Data API with 3-node DSE-next cluster
+## Stargate Data API with 3-node DSE-6.9 cluster
 
 You can start a simple Stargate configuration with the following command:
 
 ```
-./start_dse_next.sh
+./start_dse69.sh
 ``` 
 
 This convenience script verifies your Docker installation meets minimum requirements and brings up the configuration described in the `docker-compose.yml` file. The configuration includes health criteria for each container that is used to ensure the containers come up in the correct order.
 
 The convenience script uses the `-d` and `--wait` options to track the startup progress, so that the compose command exits when all containers have started and reported healthy status within a specified timeout. 
 
-The environment settings in the `.env` file include variables that describe the image tags that will be used by default, typically Data API `v1`, Stargate `v2` and DSE `4.0.7-SHA` (where `SHA` is a unique indicator for commit used to build DSE-next). The `start_dse_next.sh` script supports [options](#script-options) for overriding which image tags are used, including using a locally generated image as described [above](#building-the-local-docker-image).
+The environment settings in the `.env` file include variables that describe the image tags that will be used by default, typically Data API `v1`, Stargate `v2` and DSE `4.0.7-SHA` (where `SHA` is a unique indicator for commit used to build DSE-6.9). The `start_dse69.sh` script supports [options](#script-options) for overriding which image tags are used, including using a locally generated image as described [above](#building-the-local-docker-image).
```

Comment:
Please remove reference to Stargate image tag and change DSE image tag to 6.9 instead of 4.0.7-SHA